docs/pushmsg.md

```diff
@@ -15,6 +15,8 @@ The token is returned in the REST request and is viewable in the WebUI.
 
 Now you can simply use [curl](https://curl.haxx.se/), [HTTPie](https://httpie.org/) or any other http-client to push messages.
 
+> Since gotify/server v2.0.4 messages will be parsed as markdown.
```

Comment:
```suggestion
> Since gotify/server v2.0.4 and gotify/android v2.0.6, messages will be rendered as Markdown.
```

It would be good to add android version as well.